codebase/config/sync/views.view.solr_search_content.yml

```diff
@@ -20,11 +19,7 @@ dependencies:
     - field.storage.node.field_date_created
     - field.storage.node.field_date_published
     - field.storage.node.field_description
-    - field.storage.node.field_digital_identifier
```

Comment:
what does it mean to remove this? What effect does it have on things?

Reply:
The only change I made was to add content type sorting through the admin UI, then ran `make config-export`. I don't know why Drupal changed the rest of the config.